samples/graph-proactive-installation/nodejs/README.md

```diff
@@ -0,0 +1,69 @@
+# Proactive Installation Sample App
+
+This sample app illustartes the proactive installation of app using Graph API and sending proactive notification to users from GroupChat or Channel.
+
+Language Used : Nodejs
+
+## Prerequisites
+### Tools
+
+- [Node.js](https://nodejs.org) version 10.14 or higher
+
+    ```bash
+    # determine node version
+    node --version
+    ```
+## To try this sample
+1. Clone the repository
+    ```bash
+    git clone https://github.com/OfficeDev/microsoft-teams-samples.git
+    ```
+- In a terminal, navigate to `samples/javascript_nodejs/graph-proactive-installation`
```

Comment:
Please update the navigate path `samples/graph-proactive-installation/nodejs`

Reply:
Updated in Readme file